governance.tt

```diff
@@ -0,0 +1,139 @@
+  <div class="span8">
+    <div class="hero-unit">
+      <h2>RFC Process</h2>
+      <p>The "RFC" (request for comments) process is intended to provide a
+         consistent and controlled path for new features to enter the Nix language,
+         packages and OS, so that all community can be confident about the
+         direction the ecosystem is evolving in.
+      </p>
+      <a class="btn btn-block btn-large btn-success">Learn more</a>
+    </div>
+  </div>
+  <div class="span4">
+    <a href="#rfc-process-modal" class="thumbnail" data-toggle="modal">
+      <img src="https://raw.githubusercontent.com/NixOS/rfcs/master/rfcs/0036-review-process.png"
+           alt="rfc process"
+           />
+    </a>
+    <div id="rfc-process-modal"
+         class="modal hide fade"
+         tabindex="-1"
+         role="dialog"
+         data-backdrop="false"
+         style="width: 700px; left:45%;">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal" aria-hidden="true">×</button>
+      </div>
+      <div class="modal-body" style="max-height: 600px;">
+        <img src="https://raw.githubusercontent.com/NixOS/rfcs/master/rfcs/0036-review-process.png"
+             alt="rfc process"
+             style="width: 100%"
+             />
+      </div>
+    </div>
+  </div>
+</div>
+
+<h2>Teams</h2>
+
+<div class="row-fluid">
+  <div class="span4">
+    <h3>RFC Steering Committee</h3>
+    <p>Responsible for forming an RFC Shepherd team from the available
+       nominations on each RFC. RFC Shepherd team is then responsible for
+       accepting or rejecting a specific RFC.
+    </p>
+    <a class="btn btn-block btn-success"
+      href="[%root%]/teams/rfc-steering-committee.html">Members &amp; Contacts</a>
+  </div>
+  <div class="span4">
+    <h3>Security Team</h3>
+    <p>Security is a priority for the NixOS community. In addition to
+       community-sourced developments, NixOS has a dedicated team, to look into
+       privately reported security issues.
+    </p>
+    <a class="btn btn-block btn-success"
+       href="[%root%]/teams/security.html">Members &amp; Contacts</a>
+  </div>
+  <div class="span4">
+    <h3>Infrastructure Team</h3>
+    <p>The responsibility of the team is to provide infrastructure for the Nix and
+       NixOS community.
+    </p>
+    <a class="btn btn-block btn-success"
+       href="[%root%]/teams/infrastructure.html">Members &amp; Contacts</a>
+  </div>
+</div>
+
+<div class="row-fluid">
+  <div class="span4">
+    <h3>NixOS Release Team</h3>
+    <p>A team that twice a year manages a release of NixOS. It is responsible
+       whole release process from setting the roadmap to uploading the
+       artifacts.
+    </p>
+    <a class="btn btn-block btn-success"
+       href="[%root%]/teams/nixos_release.html">Members &amp; Contacts</a>
+  </div>
+  <div class="span4">
+    <h3>NixCon Team</h3>
+    <p>A team that helps organize NixCon, a conference, where Nix and NixOS
```

Comment:
"a conference where"

Or maybe "a conference for the Nix and NixOS community".